pageserver/src/tenant/timeline/span.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(not(debug_assertions))]
+#[inline]
+pub(crate) fn debug_assert_current_span_has_tenant_and_timeline_id() {}
+
+#[cfg(debug_assertions)]
+#[inline]
+pub(crate) fn debug_assert_current_span_has_tenant_and_timeline_id() {
+    use utils::tracing_span_assert;
```

Comment:
Since the scope is small, `use utils::tracing_span_assert::MultiNameExtractor;` could be used instead.

Reply:
Not on this PR.